hs/app/reach/Main.hs

```diff
@@ -2517,15 +2542,14 @@ support = command "support" $ info (pure g) d
           Just _ -> t `by` "error" >>= T.putStrLn . (pack "\nError while acquiring access token:\n" <>)
         exitWith $ ExitFailure 1
       gat <- unpack <$> t `by` "access_token"
-      when (null rsh) $ putStrLn "\nDidn't find index.rsh in the current directory; skipping..."
-      when (null mjs) $ putStrLn "\nDidn't find index.mjs in the current directory; skipping..."
+      let flattenedPairs = L.foldl' (<>) [] arrayOfPairs
       -- @TODO: Also add output of reach hashes!
       parseRequest "POST https://api.github.com/gists"
         >>= httpBS
           . setRequestHeader "User-Agent" [ BSI.packChars "reach" ]
           . setRequestHeader "Authorization" [ BSI.packChars ("token " <> gat) ]
           . setRequestHeader "Accept" [ BSI.packChars "application/vnd.github.v3+json" ]
-          . setRequestBodyJSON (object [ "files" .= object (rsh <> mjs) ])
+          . setRequestBodyJSON (object [ "files" .= object flattenedPairs ])
```

Comment:
So, I'd do `.... object (concat arrayOfPairs) ....`

Reply:
Thanks for the simplification.

[ba07974](https://github.com/reach-sh/reach-lang/pull/1063/commits/ba07974e7d648336f7489ead9b8718e5792e60a5)